src/pages/documentation/mobile-core/rules-engine/index.md

```diff
@@ -51,11 +51,17 @@ Conditions can be one match or a group of matches.
 
 On older versions of Experience Platform Mobile SDKs (prior to iOS version 1.6.2, Android version 1.5.4), after the first launch of the app, it always takes some time to download the rules from the remote servers. During this time, Rules Engine won't be able to evaluate the first several events until the rules are loaded. Starting from iOS version 1.6.2 and Android version 1.5.4, we add the capability to cache the events before the rules are downloaded and will re-evaluate those events afterward. This change is mainly to enable the trigger of Postback based on the install event.
 
-## Using Bundled Rules (iOS AEP 3.x Only)
+## Using Bundled Rules
+
+>[!NOTE]
+>
+>This feature is available from iOS AEP 3.x and Android 1.11.2.
```

Comment:
We should use the gatsby format for info notes here:
```
<InlineAlert variant="info" slots="text"/>
This feature is available from iOS AEP 3.x and Android 1.11.2.
```

Reply:
Replace with an InlineAlert.